client/me/account/main.jsx

```diff
@@ -492,7 +502,8 @@ const Account = React.createClass( {
 						id="url"
 						name="url"
 						onFocus={ this.recordFocusEvent( 'Web Address Field' ) }
-						valueLink={ this.valueLink( 'user_URL' ) }
+						value={ this.getUserSetting( 'user_URL' ) || '' }
+						onChange={ this.updateUserSettingInput( 'user_URL' ) }
```

Comment:
> we need to remember that there is no sense of a value comparison of functions. every time render() gets called here we create a new function closure and even if all other things are identical that won't shallow-compare to equality with the previous run

Thanks for explaining this! The old-style non-Redux Calypso code still contains a lot of performance gotchas like this. For example, the `Account` component will still use `eventRecorder` mixin at places like:
```
onFocus={ this.recordFocusEvent( 'Email Address Field' ) }
```
And all these subtle performance problems will kind of magically disappear during the Reduxification!